telegram/ext/filters.py

```diff
@@ -293,6 +293,13 @@ def filter(self, message):
 
     game = _Game()
 
+    class _SuccessfulPayment(BaseFilter):
```

Comment:
This filter seems to be duplicated as it's been added in another commit too ( see further down)